docs/source/examples/mae.rst

```diff
@@ -0,0 +1,55 @@
+.. _mae:
+
+MAE
+===
+
+Example implementation of the Masked Autoencoder (MAE) architecture. MAE is a
+transformer model based on the `Vision Transformer (ViT) <https://arxiv.org/abs/2010.11929>`_ 
+architecture. It learns image representations by predicting pixel values for
+masked patches on the input images. The network is split into an encoder and
+decoder. The encoder generates the image representation and the decoder predicts
+the pixel values from the representation. MAE outperforms other self-supervised
```

Comment:
Too much text. I'd strip the part starting with `MAE outperforms other...` as this part will be outdated soon anyway. I think a quick description of the model is fine but then we should make it consistently also for other models.

Reply:
Removed it :)